### ISSUE TITLE
Maximum file size

### ISSUE THREAD

**Issue by pizzafroide** [igorklopov/enclose#55]
The maximum embeddable file size is 12.189.591 bytes (11Mb + 639Kb + 919b) as demonstrated hereafter. This is due to a limitation in V8. As a workaround, I split files exceeding this size into embeddable chunks, and I stitch these chunks together later on. Still, it'd be interesting if this job was done by EncloseJS itself.

``` js
var fs = require('fs');
var path = require('path');
var maxSize = 11 * 1024 * 1024 + 639 * 1024 + 919;
fs.writeFileSync(path.join(__dirname, 'ok.bin'), new Buffer(maxSize));
fs.writeFileSync(path.join(__dirname, 'ko.bin'), new Buffer(maxSize + 1));
```

**Comment by igorklopov** [comment]
This turned out to be a limitation of EncloseJS. A month ago i found it and fixing it since then :) The fix is almost ready.

**Comment by igorklopov** [comment]
This limitation is fixed in EncloseJS since 1.0.0. Please try again. Thanks!

**Comment by pizzafroide** [comment]
Yep, it's working. Thanks!
